pakyow-presenter/lib/pakyow/presenter/presentable.rb

```diff
@@ -8,13 +8,27 @@ def initialize(*args)
         super
       end
 
-      def presentable(*args)
-        presentables.concat(args).uniq!
+      def presentable(name, value = nil)
+        args = {
+          name: name,
+          value: value
+        }
+
+        args[:block] = Proc.new if block_given?
```

Comment:
Making method calls from within presentable won't work in the case of defining the presentable outside of the request/response lifecycle. We discussed this on a call and decided to call methods and blocks immediately when we're using presentable from within a route, but deferring the calls to within the controller at presentation time otherwise. 

We also decided to handle the case where both a value and a block are provided by defaulting to the value unless it's nil, in which case we'll call the block and use its result as the value.